mono/mini/mini-windows.c

```diff
@@ -152,6 +152,52 @@ get_win32_restore_stack (void)
 
 	return start;
 }
+#elif HAVE_API_SUPPORT_WIN32_RESET_STKOFLW && TARGET_ARM64
+static gpointer
+get_win32_restore_stack(void)
+{
+	static guint8* start = NULL;
+	guint8* code;
+
+	if (start)
+		return start;
+
+	const int size = 128;
```

Comment:
each instruction on ARM64 is 4 bytes, so this should fit in ~44 bytes I think? Allocating 64 bytes would probably be enough.